Converted player groups to YAML (#6488) Compiling Warning

![image](https://user-images.githubusercontent.com/102368092/163675966-9c17c8be-24de-482c-adde-623fe8c826cb.png)

Why is there an error?

OS is CentOS7

corresponding link is https://github.com/rathena/rathena/commit/51ddc63a025a962dccb8d7a40996131c950a8677

So does Ubuntu 21.04

![Capture](https://user-images.githubusercontent.com/60462533/163683599-0b598b8a-b1e4-4072-ac89-0dc72050863f.PNG)
```
        CXX     instance.cpp
        CXX     intif.cpp
In file included from ../common/cbasetypes.hpp:365:0,
                 from intif.hpp:9,
                 from intif.cpp:4:
intif.cpp: In function ‘int mapif_parse_WisToGM_sub(map_session_data*, __va_list_tag*)’:
intif.cpp:1345:42: warning: ‘e_pc_permission’ is promoted to ‘unsigned int’ when passed through ‘...’
  e_pc_permission permission = va_arg(va, e_pc_permission);
                                          ^
intif.cpp:1345:42: note: (so you should pass ‘unsigned int’ not ‘e_pc_permission’ to ‘va_arg’)
intif.cpp:1345:42: note: if this code is reached, the program will abort
        CXX     itemdb.cpp
        CXX     log.cpp

same as me on centos 7

![gambar](https://user-images.githubusercontent.com/8316103/163735549-6186419d-33b1-48c7-babc-5ca0eed25eb2.png)

```
        CXX     homunculus.cpp
        CXX     instance.cpp
        CXX     intif.cpp
In file included from ../common/cbasetypes.hpp:365:0,
                 from intif.hpp:9,
                 from intif.cpp:4:
intif.cpp: In function ‘int mapif_parse_WisToGM_sub(map_session_data*, __va_list_tag*)’:
intif.cpp:1345:42: warning: ‘e_pc_permission’ is promoted to ‘unsigned int’ when passed through ‘...’
  e_pc_permission permission = va_arg(va, e_pc_permission);
                                          ^
intif.cpp:1345:42: note: (so you should pass ‘unsigned int’ not ‘e_pc_permission’ to ‘va_arg’)
intif.cpp:1345:42: note: if this code is reached, the program will abort
        CXX     itemdb.cpp
        CXX     log.cpp
        CXX     mail.cpp
```

--- a/src/map/intif.cpp
+++ b/src/map/intif.cpp
@@ -1342,7 +1342,7 @@ int intif_parse_WisEnd(int fd)
  */
 static int mapif_parse_WisToGM_sub(struct map_session_data* sd,va_list va)
 {
-	e_pc_permission permission = va_arg(va, e_pc_permission);
+	e_pc_permission permission = static_cast<e_pc_permission>(va_arg(va, uint32));
 	char *wisp_name;
 	char *message;
 	int len;

